Documentation/README_SystemKeyboard.md

```diff
@@ -36,4 +36,4 @@ private void Update()
 
 ## System keyboard example
 
-You can see a simple example of how to bring up system keyboard in [`MixedRealityKeyboard.cs`](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/MixedRealityKeyboard.cs)
+You can see a simple example of how to bring up system keyboard in `MixedRealityKeyboard.cs` (Assets/MRTK/SDK/Experimental/Features/UX/MixedRealityKeyboard.cs)
```

Comment:
xref?

Reply:
i'll check to see if there is an api doc page for this class

Reply:
this is actually pointing to the script as an example. not sure what the value of pointing to the API docs for the class will provide.